domain/application/service/application.go

```diff
@@ -82,21 +82,22 @@ type ApplicationState interface {
 	// error satisfying [applicationerrors.UnitNotFound] if the unit doesn't exist.
 	SetUnitWorkloadStatus(context.Context, coreunit.UUID, *application.StatusInfo[application.WorkloadStatusType]) error
 
-	// GetUnitCloudContainerStatus returns the cloud container status of the specified
+	// GetUnitK8sPodStatus returns the cloud container status of the specified
 	// unit, returning an error satisfying [applicationerrors.UnitNotFound] if the unit
-	// doesn't exist.
-	GetUnitCloudContainerStatus(context.Context, coreunit.UUID) (*application.StatusInfo[application.CloudContainerStatusType], error)
+	// doesn't exist, or an error satisfying [applicationerrors.UnitStatusNotFound] if
+	// the status is not set.
+	GetUnitK8sPodStatus(context.Context, coreunit.UUID) (*application.StatusInfo[application.CloudContainerStatusType], error)
```

Comment:
Don't rename this yet.

I want to rename in one shot, otherwise it's confusing.

Reply:
Cool no worries, I'll drop the commit